firmware/modes/programmer/at89/at89.c

```diff
@@ -0,0 +1,654 @@
+#include "at89.h"
+
+static zif_bits_t zbits_null = {0, 0, 0, 0, 0};

```

Comment:
bitmasks should probably be written in hex for clarity

Reply:
Agreed, will fix next commit.